Extensions/old_blue.js

```diff
@@ -64,17 +64,10 @@ XKit.extensions.old_blue = new Object({
 						--gray-13: rgba(var(--rgb-black), 0.13);
 						--gray-7: rgba(var(--rgb-black), 0.07);
 
-						${this.preferences.old_font.value ?
-							'--font-family: "Helvetica Neue", "HelveticaNeue", Helvetica, Arial, sans-serif'
-							: ''
-						}
+						${this.preferences.old_font.value ? '--font-family: "Helvetica Neue", "HelveticaNeue", Helvetica, Arial, sans-serif' : ''}
```

Comment:
Not in scope of this PR but this and its sibling below would be better factored out as a variable instead of a ternary in a `${}`